test/array.test.js

```diff
@@ -530,3 +530,86 @@ buildTest({
   largeArraySize: 2e4,
   largeArrayMechanism: 'default'
 })
+
+test('error on invalid value for largeArraySize /1', (t) => {
```

Comment:
I would use a `for-loop` to generate test-case.
Split in multiple one with same structure is not good for expanding when we need.

Reply:
I would like to keep it simple. I am just trying to improve test coverage.